packages/core/src/types/v3/people/get-combined-credits.ts

```diff
@@ -1,27 +1,14 @@
+import {PeopleEndpointBaseCrewGetMovieGetCombinedCredit, PeopleEndpointBaseCastGetMovieGetCombinedCredit} from "./bast-cast-crew-credits";
+
 export interface PeopleGetCombinedCreditsResponse {
   cast: PeopleGetCombinedCreditsCast[];
-}
-
-export interface PeopleGetCombinedCreditsCast {
+  crew: PeopleGetCombinedCreditsCrew[];
   id: number;
-  original_language: string;
-  episode_count: number;
-  overview: string;
-  origin_country: string[];
-  original_name: string;
-  genre_ids: number[];
-  name: string;
-  media_type: string;
-  poster_path: string;
-  first_air_date: string;
-  vote_average: number;
-  vote_count: number;
-  character: string;
-  backdrop_path: string;
-  popularity: number;
-  credit_id: string;
 }
 
+export type PeopleGetCombinedCreditsCast = PeopleEndpointBaseCastGetMovieGetCombinedCredit & {media_type: string};
```

Comment:
Could we type media_type better? I think we have "movie" | "tv" and "all" | "movie" | "tv" | "person" elsewhere

Reply:
Taken care of.